packages/app/src/specs/SpecFileItem.spec.tsx

```diff
@@ -1,10 +1,25 @@
 import SpecFileItem from './SpecFileItem.vue'
 
 describe('SpecFileItem', () => {
-  it('should render SpecFileItem', () => {
+  it('renders a SpecFileItem with the base filename and extension having different highlights when not selected', () => {
     cy.mount(() =>
       (<div class="bg-gray-1000 group">
         <SpecFileItem fileName="Hello" extension=".spec.jsx" selected={false}></SpecFileItem>
       </div>))
+
+    cy.contains('Hello.spec.jsx').should('be.visible')
```

Comment:
Haven't used percy much, is it possible to tell it in some way that what we care about in the snapshot is that the colors of `hello` and `.spec.jsx` are not the same? Should we add an assertion before the snapshot to check `getComputedStyle` on the two elements make sure that 

`renders a SpecFileItem with the base filename and extension having different highlights when not selected`

specifically is true?

Maybe overkill since that is one of the changes that Percy should alert us to.